content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -212,9 +211,8 @@ For further information, see [Managing the kubeadm drop-in file for systemd](/do
 
 ### Use kubeadm with CRI runtimes
 
-Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
-The container runtime used by default is Docker, which is enabled through the built-in
-`dockershim` CRI implementation inside of the `kubelet`.
+The container runtime used in kubeadm by default is Docker, which is enabled through the
```

Comment:
I'm not sure this is entirely true because kubeadm auto-detects the CRI, so it doesn't work by default only with docker.
@rosti opinions?

Reply:
removing this section as it is not needed.
CRI detection is outlined in https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime